### PR TITLE
Port 'dcos marathon ping' in Go.

### PR DESCRIPTION
**Pull request by armandgrillet**
Adresses https://jira.mesosphere.com/browse/DCOS-60888

### CODE REVIEW

File: pkg/cmd/marathon/marathon_ping.go [hunk 1]
```diff
@@ -13,7 +15,19 @@ func newCmdMarathonPing(ctx api.Context) *cobra.Command {
...
+			if err != nil {
+				return err
+			}
+
+			_, err = client.API.Ping()
+			if err != nil {
+				return err
+			}
+
+			// If no err, a pong has been received.
```
Comment (janisz) [inline pkg/cmd/marathon/marathon_ping.go:30]: Why not use actual response body?

**Review by janisz — COMMENTED** [review]
Can you add a unit test?

File: pkg/cmd/marathon/marathon_ping.go [hunk 2]
```diff
@@ -13,7 +15,19 @@ func newCmdMarathonPing(ctx api.Context) *cobra.Command {
...
+			fmt.Fprintf(ctx.Out(), "Marathon ping response[5x]: \"pong\"\n")
```
Comment (janisz) [inline pkg/cmd/marathon/marathon_ping.go]:
```suggestion
			fmt.Fprintf(ctx.Out(), "Marathon ping response[1x]: \"pong\"\n")
```

File: pkg/cmd/marathon/marathon_ping.go (hunk 1 above)
Comment (armandgrillet) [reply pkg/cmd/marathon/marathon_ping.go:30]: The number appearing in `Marathon ping response[%dx]` is the number of times the ping requests was sent according to the Python logic.

**Review by makkes — APPROVED** [review]
lgtm (except the linter errors 😉)

**Comment by armandgrillet** [comment]
Unit test added and linting passing.

**Comment by armandgrillet** [comment]
@mesosphere-ci run mac integration tests

**Comment by drewvanstone** [comment]
Mind adding a description to this PR?

File: pkg/cmd/marathon/marathon_ping.go [hunk 3]
```diff
@@ -22,3 +38,27 @@ func newCmdMarathonPing(ctx api.Context) *cobra.Command {
...
+		if err != nil {
+			return 0, err
+		}
+		numberOfMasters = len(masters)
+	}
+
+	for i := 1; i <= numberOfMasters; i++ {
+		_, err := client.API.Ping()
+		if err != nil {
+			return 0, fmt.Errorf("Unable to ping leading Marathon master %d time(s)", i)
```
Comment (drewvanstone) [inline pkg/cmd/marathon/marathon_ping.go]: I've been seeing errors returned without capitalization in our other projects. @makkes did we decide that error messages should be uncapitalized?

**Review by drewvanstone — COMMENTED** [review]
LGTM other than one comment and lack of PR description.

File: pkg/cmd/marathon/marathon_ping.go (hunk 3 above)
Comment (armandgrillet) [reply pkg/cmd/marathon/marathon_ping.go]: I can see that we mostly do not capitalize the first letter in dcos-core-cli but we have some inconsistencies, removing the capital now.

**Review by drewvanstone — APPROVED** [review]

**Review by janisz — APPROVED** [review]

**Comment by armandgrillet** [comment]
@mesosphere-ci run windows integration test

**Comment by armandgrillet** [comment]
@mesosphere-ci run windows integration test

**Comment by armandgrillet** [comment]
@mesosphere-ci run windows integration tests

**Comment by armandgrillet** [comment]
@mesosphere-ci run windows integration tests

**Pull request merged**